blockchain/blockchain.go

```diff
@@ -320,7 +320,13 @@ func (bc *BlockChain) migrateState(rootHash common.Hash) error {
 	srcCachedDB := bc.StateCache().TrieDB()
 	targetDB := statedb.NewDatabase(&stateTrieMigrationDB{bc.db})
 
-	trieSync := state.NewStateSync(rootHash, targetDB.DiskDB())
+	// TODO-Klaytn Change NewMemDB to real targetDB for restarting state migration
+	// Present partitionedDB doesn't not support iterator. Therefore targetDB can't not be used.
+	// If state migration is finished without restarting node, this fake empty DB is ok.
```

Comment:
```suggestion
	// Present bloom filter for migration.
	// Since iterator doesn't support partitionedDB, we cannot use targetDB.
	// If state migration is finished without restarting node, this fake empty DB is ok.
```